--- orig/src/eh-printf.h
+++ tidy/src/eh-printf.h
@@ -22,9 +22,11 @@ License (COPYING) along with this library; if not, see:
 #ifndef EH_PRINTF
 #define EH_PRINTF
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>
 #include <stdarg.h>
@@ -37,8 +39,10 @@ int eh_vprintf(const char *format, va_list ap);
 
 int eh_vsnprintf(char *str, size_t size, const char *format, va_list ap);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EH_PRINTF */
--- orig/src/eh-sys-context.h
+++ tidy/src/eh-sys-context.h
@@ -22,9 +22,11 @@ License (COPYING) along with this library; if not, see:
 #ifndef EH_SYS_CONTEXT
 #define EH_SYS_CONTEXT
 
+/*
 #ifdef __cplusplus
 extern "C" {
 #endif
+*/
 
 #include <stddef.h>
 
@@ -36,8 +38,10 @@ size_t eh_sys_output_char(void *ctx, char c);
 
 size_t eh_sys_output_str(void *ctx, char *buf, size_t len);
 
+/*
 #ifdef __cplusplus
 }
 #endif
+*/
 
 #endif /* EH_SYS_CONTEXT */
